### PR TITLE
Publish local

### PR DESCRIPTION
**Pull request by cvogt**

### CODE REVIEW

**Comment by cvogt** [comment]
FYI, if you want to add CBT's maven cache to SBT's resolvers, this does the job

`resolvers += "CBT Maven Cache" at "file://"+Path.userHome.absolutePath+"/<path-to-cbt>/cbt/cache/maven"`

File: build/build.scala [hunk 1]
```diff
@@ -13,4 +13,19 @@ class Build(val context: Context) extends BaseBuild{
...
+  def groupId: String = "org.cvogt"
+
+  def defaultVersion: String = "0.1"
+  def name: String = "cbt"
+
+  // Members declared in cbt.Publish
+  def description: String = "Fast, intuitive Build Tool for Scala"
+  def developers: Seq[cbt.Developer] = Nil
+  def inceptionYear: Int = 2016
+  def licenses: Seq[cbt.License] = Seq( License.Apache2 )
```
Comment (MasseGuillaume) [inline build/build.scala]: really nice to see something sane like this :)

File: build/build.scala [hunk 2]
```diff
@@ -13,4 +13,19 @@ class Build(val context: Context) extends BaseBuild{
...
+  def organization: Option[cbt.Organization] = None
+  def scmConnection: String = ""
```
Comment (MasseGuillaume) [inline build/build.scala]: `Option[String]` ?

Comment (cvogt) [inline build/build.scala]: eventually , yeah :)

**Pull request merged**